UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/HomeController.cs

```diff
@@ -21,10 +24,14 @@ public IActionResult Index()
             return View();
         }
 
-        [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+        [Route("/error")]
         public IActionResult Error()
         {
-            return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
+            string correlationId = Convert.ToString(GetCurrentCorrelationId());
```

Comment:
GetCurrentCorrelationId returns string only so is it required to explicitly convert to string again?

Reply:
Removed explicit conversion